usvm-python/usvm-python-runner/build.gradle.kts

```diff
@@ -0,0 +1,41 @@
+import usvmpython.MANUAL_RUN_GROUP_NAME
+import usvmpython.MANUAL_TEST_FOR_RUNNER
+import usvmpython.MANUAL_TEST_FOR_RUNNER_ENTRY
+
+plugins {
+    id("usvm.kotlin-conventions")
+    `maven-publish`
+}
+
+dependencies {
+    implementation(project(mapOf("path" to ":usvm-python:usvm-python-commons")))

```

Comment:
Why do we use `"path"` here? Also, can't you use `USVM_PYTHON_RUNNER_MODULE` from `Names.kt`, can you?

Reply:
First part: above, second part: done.